docs/_documentations/overview.md

```diff
@@ -9,21 +9,72 @@ type: document
+
+6. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window, 'problems' and 'output' panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
+
+7. **Support for your IDE's debug features**<sup>*</sup>  Full compatiility with the debug functionality of your IDE.
+
+8. **Visualise your container application's footprint such as CPU and memory usage** <sup>*</sup> Codewind's Application Metrics Dashboard exposes your microservice's core metrics, including HTTP(S) request and throughput information, as well as CPU usage, memory footprint and heap.
+
+9. **Integrated load-testing** <sup>*</sup> Configure and run load tests against your microservices using [Codewind's performance dashboard](https://www.youtube.com/watch?v=nfJt3f5TUvc). Use the graphs to compare your repeated load test results and iterate on performance improvements. 
+
+<sup>*</sup> where supported.
+
+## What Integrated Development Environments (IDEs) does Codewind support?
+
+* Codewind is available as an extension for [**VS Code**](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) and [**Eclipse**](https://marketplace.eclipse.org/content/codewind) for local development work. In this setup, your integrated development environment is located locally, but the build and run itself can be done either locally or remotely.
+
+* If instead you prefer to develop your code from within the cloud enviroment where your microservice will be built and run, you can use Codewind for [**Eclipse Che**](https://www.eclipse.org/codewind/mdt-che-installinfo.html).
+
+## What cloud-native technology does Codewind support?
+When it comes to running your containerised application on the cloud, Codewind supports [Kubernetes](https://kubernetes.io/) and [Openshift](https://www.openshift.com/) for container orchestration. 
+
+## How does Codewind support my development architecture?
+Codewind supports several architectures, so you can use your local, test and deployment infrastructure however suits your teams. 
+
+* **Support for hosting your development environment locally or remotely** Your code development, which takes place within your IDE, can be located either locally, or on your cloud. 
+
+* **Support for building and running your containers locally or remotely** The building and running of your microservice is actioned by the Codewind Server, which can be located locally or on your cloud.
+
+This means that Codewind can be used in 3 different setups:
+
+1. **local configuration** - your IDE is hosted on your desktop. The building and running of your microservice also takes place on your desktop, via your local Docker Desktop installation.
```

Comment:
- Capitalize the first word of the sentence for all three of these sentences on lines 60, 62, and 64.
- Comma not needed after "desktop."

Reply:
done